packages/components/src/SmartObjectFunction.tsx

```diff
@@ -90,10 +93,12 @@ export const SmartObjectFunction = ({
       const res = await computer.query({ ids: [smartObject._id] })
       setFunctionResult({ _rev: res[0] })
       setModalTitle("Success!")
+      showLoader(false)
```

Comment:
I know this is super minor but it would be a little nicer if we could move the showLoader(false) under the try-catch block.